### ISSUE TITLE
SkinFiles Folder Empty

### ISSUE THREAD

**Issue by Varache** [jilleb/mib1-toolbox#2]
Hello, 

First, thank you for this great work! It's impressive.

I installed MIB1 TOOLBOX and run dump_all script but I don't have any data in the SkinFiles folder.
Is there a specific procedure?

Thank you in advance

**Comment by jilleb** [comment]
It will not dump to SkinFiles but to the dump folder.

**Comment by Varache** [comment]
Sorry I misspoke. I was talking about the SkinFiles folder in the Dump folder.
![Screen Shot 05-27-20 at 10 52 PM](https://user-images.githubusercontent.com/65035282/83071063-c75e9c80-a06c-11ea-8e9c-225090bc8bb9.PNG)

**Comment by jilleb** [comment]
I fixed this bug in b88ceeceb2828c01929fc4f2618dea372d71c76b, are you sure you're using the latest version?

**Comment by Varache** [comment]
I think yes. I had checked in the script of my SD card.
![Screen Shot 05-27-20 at 10 58 PM](https://user-images.githubusercontent.com/65035282/83071569-a185c780-a06d-11ea-82c5-98d710f04f65.PNG)

**Comment by jilleb** [comment]
Ahh, it's Audi, I see... Audi has a slightly different folder structure. I'll make a script that writes the entire folder tree to SD, that makes it easier for me and others to write scripts

**Comment by Varache** [comment]
Yes it's AUDI. 
Thank you so much!

**Comment by jilleb** [comment]
👍 
No problem!

I added a quick and dirty dirlisting script in the latest commit. Can you try it?
The script will probably never end, unless you unplug the SD-card, but it will write the output of ls to the SD-card.

**Comment by Varache** [comment]
OK, I try tomorrow when I'm in the car. Thank you!

**Comment by Varache** [comment]
Well I tried because I didn't want to wait ;-)
I have the impression that the script does not start?

![IMG_20200528_002931](https://user-images.githubusercontent.com/65035282/83078914-f845ce00-a07a-11ea-9059-5c752d9dc5f6.jpg)
![IMG_20200528_002853](https://user-images.githubusercontent.com/65035282/83078926-fda31880-a07a-11ea-9289-0ca55e71a83b.jpg)
![IMG_20200528_003020](https://user-images.githubusercontent.com/65035282/83078932-01369f80-a07b-11ea-9d8d-47957ddf1934.jpg)

**Comment by jilleb** [comment]
hmmmm strange... that's really weird

**Comment by Varache** [comment]
I tried to delete the (original) dump_all.sh script from the SD card.
 And I replaced the script name dirlisting.sh with dump_all.sh.

![Screen Shot 05-28-20 at 11 08 AM](https://user-images.githubusercontent.com/65035282/83122619-328f8980-a0d4-11ea-9cc1-4c785a6e1af0.PNG)

Then I tried to launch the script from the first command.
This is the dump_all.sh (original) script that launched when I deleted it from the SD card.
The scripts are not read on the SD card ??

**Comment by Varache** [comment]
In addition? I replayed a complete installation with the configuration described in the previous post.
After installation, when I launch the script, the console still displays no data.
I hope to be clear in the handling explanations.

**Comment by Varache** [comment]
Hello,
I tried to comment out the cd / line and the script starts.

![Screen Shot 05-29-20 at 04 36 PM](https://user-images.githubusercontent.com/65035282/83271828-aa3fe000-a1ca-11ea-81b9-4b44fb9f75f0.PNG)

I got the dirlisting.txt file.
I hope it is what you expected.
Thank you.

The file will be accessible at the following address: http://dl.free.fr/tn9nAbWUN

**Comment by Varache** [comment]
Hello,
In the attached file I do not find a resource folder, nor any * .mif files.
Could you just guide me? That I try to do things.
Thank you in advance.

**Comment by Varache** [comment]
Hello,
I try to progress
I found a kzb file. It is an extension of UI design software - Rightware Kanzi®. I used Dragon Unpacker to try to see images inside the kzb file. I got pictures.
I don't know if it's an interesting track or not.

![content_kzb-0000001305](https://user-images.githubusercontent.com/65035282/84576672-e2dcdd80-adb6-11ea-96c1-128b8d32c6c3.png)
![content_kzb-0000000792](https://user-images.githubusercontent.com/65035282/84576676-e40e0a80-adb6-11ea-81df-e9628f35c491.png)

**Comment by jilleb** [comment]
Yes, the kanzi files are the archives where Audi keeps the graphics. The 2nd picture is the texture map they use on 3d models of the car.

**Comment by Varache** [comment]
Thank you for the information.

Are these the files I need to modify?Or should I look elsewhere?

Thank you in advance

**Comment by FPSUsername** [comment]
Hey @Varache, how did you unpack the kzb file with dragon unpacker? It doesn't seem to work on my end with [v5.7.0 Beta](https://sourceforge.net/projects/dragonunpacker/files/Dragon%20UnPACKer/Unstable%20releases/5.7.0%20Beta/).

I copied `content.kzb` from the `/mnt/app/eso/` directory.

The script has an incorrect path for the skinfiles for Audi, on **K1549** (latest MIB 1 High firmware), the folder `Resources` doesn't exist. All the skinfiles are placed directly in the `lsd` folder.

Creating a dirlist takes a very long time, I let it run for 30 minutes and the .txt file is 178mb in size. I just cut it off eventually.
I compressed the file as 7z and put it in a zip to upload it here. Use [7-zip](https://www.7-zip.org) (or another tool that can extract `.7z` and `.zip`) to extract the archive.
[dirlisting_MHIG_EU_AU_K1549_Incomplete](https://github.com/jilleb/mib1-toolbox/files/8334596/dirlisting_MHIG_Incomplete.zip)

**Comment by FPSUsername** [comment]
I extracted the files, apparently I had to select format options in the unpacker software.
I don't really think that the `content.kzb` file is too useful, unless the dragon unpacker can't fully unpack the images. All I found are textures for the cabrio (as shared above) and some PHEV pictures (besides UI stuff). I'm looking for the actual texture of the car you see in the car settings (for me that'll be a sedan).
I've seen the 3D model replacements on the MIB2 units, just wondering how to do that on MIB1 (but first I'm only interested in texture modding).
